packages/@webex/webex-core/src/lib/credentials/credentials.js

```diff
@@ -249,6 +249,24 @@ const Credentials = WebexPlugin.extend({
     });
   },
 
+  /**
+   * Convenience function to tell whether the user is unverified guest
+   * @returns {Boolean} if user is unverified guest or not
+   * @public
+   * @memberof Credentials
+   */
+  isUnverifiedGuest() {
+    // @ts-ignore - fix type
+    const accessToken = this.webex.credentials.supertoken.access_token;
```

Comment:
This function is in the credentials plugin, so you don't need to reference this.webex, you can just write
```js
this.supertoken.access_token
```